freqtrade/strategy/strategy_helper.py

```diff
@@ -1,11 +1,13 @@
+from typing import Union
+
 import pandas as pd
 
 from freqtrade.exchange import timeframe_to_minutes
 
 
 def merge_informative_pair(dataframe: pd.DataFrame, informative: pd.DataFrame,
                            timeframe: str, timeframe_inf: str, ffill: bool = True,
-                           append_timeframe: bool = True,
+                           suffix: Union[bool, str] = True,
```

Comment:
well - no.

Please do this in a separate PR. it's not related to this change - and this PR is already way too large.

For that separate pr please consider:
This is a user facing interface - your change is breaking all strategies currently using this.